### ISSUE TITLE
Question about test results. Local reproduction shows -2.3 gaps in PDMS.

### ISSUE THREAD

**Issue by HatakeKiki** [BraveGroup/DriveVLA-W0#15]
Hi, thank you for open-sourcing your great work. I'm trying to reproduce the model Emu3_Flow_Matching_Action_Expert_PDMS_87.2, starting with local inference on the model/vq index files/pkl files downloaded. But the final average score is 84.9 , which is inferior to the score as indicated in the model name, 87.2. 

Could you please help me to locate the cause for the score gap? It'll be great if I can have access to your csv results. Thank you in advance~

**Output of evaluation script as below:** 

```
        Finished running evaluation.
            Number of successful scenarios: 12146.
            Number of failed scenarios: 0.
            Final average score of valid results: 0.8493058339444204.
            Results are stored in: DriveVLA-W0-main/logs/navsim_eval/train_navsim_pi0_vava_from_nuplan_ft/json_output_cursor_clean/2026.03.26.07.57.02/2026.03.26.08.02.33.csv.

```

**Command used in infer and eval:**
```
bash inference/vla/eval_navsim_metric_from_json.sh
bash inference/vla/infer_navsim_flow_matching_PDMS_87.2.sh
```

**Variables in infer scripts:**
```
# ============================================================================
# 配置区域：在这里设置所有路径和参数
# ============================================================================

# 模型和配置路径
export VLA_ACTION_TOKENIZER="${VLA_ACTION_TOKENIZER:-pretrained_models/fast}"
export VLA_VLM_MODEL="${VLA_VLM_MODEL:-models/Emu3_NuPlan_Pretrain_Cktps}"
export VLA_NORM_STATS="${VLA_NORM_STATS:-configs/normalizer_navsim_test/norm_stats.json}"
export VLA_TOKEN_YAML="${VLA_TOKEN_YAML:-inference/navsim/navsim/navsim/planning/script/config/common/train_test_split/scene_filter/navtest.yaml}"

# 推理参数（可通过环境变量覆盖）
export EMU_HUB="${EMU_HUB:-models/Emu3_Flow_Matching_Action_Expert_PDMS_87.2}"
export OUTPUT_DIR="${OUTPUT_DIR:-logs/train_navsim_pi0_vava_from_nuplan_ft/json_output_cursor_clean}"
export TEST_DATA_PKL="${TEST_DATA_PKL:-data/others/navsim_emu_vla_256_144_test_pre_1s.pkl}"

# 可选参数
export VLA_NUM_WORKERS="${VLA_NUM_WORKERS:-2}"
export VLA_BATCH_SIZE="${VLA_BATCH_SIZE:-1}"

# Anchor 相关路径（用于模型内部，可通过环境变量覆盖）
export VLA_ANCHOR_CLUSTER_PATH="${VLA_ANCHOR_CLUSTER_PATH:-data/others/cluster_centers_8192.npy}"
export VLA_ANCHOR_METRIC_SCORE_PATH="${VLA_ANCHOR_METRIC_SCORE_PATH:-data/others/formatted_pdm_score_8192.npy}"



```

**Comment by HatakeKiki** [comment]
Update: I change the norm_stats.json to configs/normalizer_navsim_trainval/norm_stats.json and get higher score, 86.9, which lags behind the announced score, 87.2, a little bit.
